conans/test/functional/command/info_command_test.py

```diff
@@ -118,3 +120,19 @@ def info_build_test(self):
         self.clients["H3"].run("info . --build outdated")
         self.assert_last_line(self.clients["H3"],
                               "H0/0.1@lu/st, H1a/0.1@lu/st, H2a/0.1@lu/st, H2c/0.1@lu/st")
+
+    def test_json_output_build(self):
+        """ Test that the output of 'conan info --build' to json file is correct """
+        json_file = os.path.join(temp_folder(), "output.json")
+        json_output = '["H0/0.1@lu/st", "H1a/0.1@lu/st", "H1c/0.1@lu/st", ' \
+                      '"H2a/0.1@lu/st", "H2c/0.1@lu/st"]'
+
+        self.clients["H3"].run("info . --build missing --json=\"{}\"".format(json_file))
```

Comment:
Should this be outputing the whole info data and not only build order?

Reply:
It is the expected output (in json format)

```
  -b [BUILD], --build [BUILD]
                        Given a build policy, return an ordered list of
                        packages that would be built from sources during the
                        install command
```